### PR TITLE
hunspell: 1.7.0 -> 1.7.1

### PR DESCRIPTION
**Pull request by algorithmiker**
###### Description of changes

changelog:
https://github.com/hunspell/hunspell/releases/tag/v1.7.1

fixes CVE-2019-16707: https://nvd.nist.gov/vuln/detail/CVE-2019-16707
(this was previously patched in nixpkgs, now upstream fixed it)

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/hunspell/default.nix [hunk 1]
```diff
@@ -11,18 +18,13 @@ stdenv.mkDerivation rec {
...
+  outputs = ["bin" "dev" "out" "man"];
 
-  buildInputs = [ ncurses readline ];
-  nativeBuildInputs = [ autoreconfHook ];
+  buildInputs = [ncurses readline];
+  nativeBuildInputs = [autoreconfHook];
 
   patches = [
     ./0001-Make-hunspell-look-in-XDG_DATA_DIRS-for-dictionaries.patch
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/hunspell/default.nix:21]: Please remove the unused fetchpatch from inputs.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please revert all the formatting changes. They do not follow the nixpkgs standard.

**Comment by riceicetea** [comment]
I am sorry. I left Alejandra on and didn't notice :/ will fix.

**Comment by riceicetea** [comment]
New PR: https://github.com/NixOS/nixpkgs/pull/188972

**Comment by SuperSandro2000** [comment]
Why did you do a new PR?

**Comment by riceicetea** [comment]
Is it a big deal? It was easier to revert the changes that way.

**Comment by SuperSandro2000** [comment]
yes, in the future please force push PRs. Otherwise (review) comments might get lost and it is harder to follow the changes.

**Pull request closed without merging**